sql-statements/sql-statement-import-into.md

```diff
@@ -22,7 +22,7 @@ For TiDB Self-Hosted, `IMPORT INTO` supports importing data from files stored in
 
 ## Restrictions
 
-- For TiDB Self-Hosted, `IMPORT INTO` supports importing data within 10 TiB. For [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `IMPORT INTO` supports importing data within 50 GiB.
+- For TiDB Self-Hosted, `IMPORT INTO` supports importing data within 10 TiB. If you enable the [Global Sort](/tidb-global-sort.md) feature, `IMPORT INTO` supports importing data within 100 TiB. For [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `IMPORT INTO` supports importing data within 50 GiB.
```

Comment:
```suggestion
- For TiDB Self-Hosted, `IMPORT INTO` supports importing data within 10 TiB. If you enable the [Global Sort](/tidb-global-sort.md) feature, `IMPORT INTO` supports importing data within 10 TiB. For [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `IMPORT INTO` supports importing data within 40 GiB.
```

Reply:
The specific size limit for TiDB Dedicated is TBD now and will be udpated once it is confirmed.